components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -14,15 +14,16 @@ import * as Utils from 'utils/utils';
 export default class PostProfilePicture extends React.PureComponent {
     static propTypes = {
         compactDisplay: PropTypes.bool.isRequired,
-        enablePostIconOverride: PropTypes.bool.isRequired,
-        hasImageProxy: PropTypes.bool.isRequired,
+        enablePostIconOverride: PropTypes.bool,
+        hasImageProxy: PropTypes.bool,
         isBusy: PropTypes.bool,
         isRHS: PropTypes.bool,
         post: PropTypes.object.isRequired,
         status: PropTypes.string,
         user: PropTypes.object,
         isBot: PropTypes.bool,
         postIconOverrideURL: PropTypes.string,
+        userId: PropTypes.string
```

Comment:
Direct quote from [Abdusabri](https://github.com/abdusabri)'s PR ([link](https://github.com/mattermost/mattermost-webapp/pull/5351#discussion_r410745361)):
> Was provided to the component when used in `rhs_card`, I've added it as optional to please the TypeScript compiler :slightly_smiling_face:

Reply:
It doesn't appear to be used in this component though, so we should remove it and all references to it instead.